### PR TITLE
Illusive translations

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Partial translation. More later. Just merge these now.

### CODE REVIEW

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -860,51 +899,63 @@ msgstr ""
...
 "Помилка: --auto-sort (автосортування) пройшло, але параметр --game не було "
 "вказано."
 
-#: src/gui/state/loot_state.cpp:133
+#. translators: --game and --game-path are command-line arguments and
+#. shouldn't be translated.
+#: src/gui/state/loot_state.cpp:121
+msgid "Error: --game-path was passed but no --game parameter was provided."
+msgstr ""
+"Помилка: --auto-sort (автосортування) пройшло, але параметр --game не було "
```
Comment (Ortham) [inline resources/l10n/uk_UA/LC_MESSAGES/loot.po:915]: This looks like a copy/paste mistake.

**Review by Ortham — CHANGES_REQUESTED** [review]

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po (hunk 1 above)
Comment (IllusiveMan196) [reply resources/l10n/uk_UA/LC_MESSAGES/loot.po:915]: In my version it is there, anyhow yes it should contain "не було вказано"

Comment (IllusiveMan196) [reply resources/l10n/uk_UA/LC_MESSAGES/loot.po:915]: Poedit just adds it on new line for some reason

Comment (IllusiveMan196) [reply resources/l10n/uk_UA/LC_MESSAGES/loot.po:915]: I'll add it in new commit as well (probably it fixes whatever poedit did wrong or maybe it doesn't)

Comment (Ortham) [reply resources/l10n/uk_UA/LC_MESSAGES/loot.po:915]: That's fine, but `--auto-sort` is not `--game-path`.

**Comment by IllusiveMan196** [comment]
There it is all completed. Should be as fresh as new template file allows (Poedit still managed to split in separate lines some things (mostly new-ish longer lines)).

**Comment by IllusiveMan196** [comment]
Now it's all correct and as it should be (yes, copy-paste without much attention). Safe to merge (probably).

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po (hunk 1 above)
Comment (IllusiveMan196) [reply resources/l10n/uk_UA/LC_MESSAGES/loot.po]: Corrected in new version, sorry, didn't notice.

**Comment by Ortham** [comment]
Thanks 👍

**Pull request merged**